contrib/eigen/template.py

```diff
@@ -0,0 +1,17 @@
+pkgname = "eigen"
+pkgver = "3.4.0"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = ["cmake", "ninja", "pkgconf"]
+pkgdesc = "C++ template library for linear algebra"
+maintainer = "shtayerc <david.murko@mailbox.org>"
+license = "BSD-3-Clause"
```

Comment:
more like `MPL-2.0 AND BSD-3-Clause AND Minpack AND Apache-2.0`, though idk where each is used, but just go with that